catalog/transformer/catalog-transformer-csv-common/pom.xml

```diff
@@ -81,7 +81,7 @@
                                         <limit implementation="org.codice.jacoco.LenientLimit">
                                             <counter>COMPLEXITY</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.81</minimum>
+                                            <minimum>0.82</minimum>
```

Comment:
did you mean to lower the threshold instead of raise it? seems like that could be why your build is failing

Reply:
Oops - just grabbed the values from the cherrypick - fixed now. Thanks!